### ISSUE TITLE
Module @slidev/types missing when using the globally installed slidev

### ISSUE THREAD

**Issue by christian-nils** [slidevjs/slidev#189]
**Describe the bug**
After installing globally slidev, running `slidev` in a repository with only the file `slides.md` returns an error:

```bash
(base) [christian-nils@als-of-wl2912 slidevv]$ slidev
node:internal/modules/cjs/loader:941
  const err = new Error(message);
              ^

Error: Cannot find module 'packages/theme-seriph/node_modules/@slidev/types'
Require stack:
- /home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/theme-default/setup/windicss.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:941:15)
    at Function.resolve (node:internal/modules/cjs/helpers:99:19)
    at m (/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/cli/node_modules/jiti/dist/jiti.js:1:52899)
    at f (/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/cli/node_modules/jiti/dist/jiti.js:1:53712)
    at /home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/theme-default/setup/windicss.ts:2:14
    at f (/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/cli/node_modules/jiti/dist/jiti.js:1:54601)
    at loadSetups (/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/cli/dist/chunk-J2KFKCYM.js:116:32)
    at createWindiCSSPlugin (/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/cli/dist/chunk-J2KFKCYM.js:137:18)
    at ViteSlidevPlugin (/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/cli/dist/chunk-J2KFKCYM.js:1175:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/christian-nils/.nvm/versions/node/v16.2.0/lib/node_modules/@slidev/theme-default/setup/windicss.ts'
  ]
}
```

**To Reproduce**
1. `npm i -g @slidev/cli`
2. `touch slides.md`
3. `slidev`

**Desktop (please complete the following information):**
 - OS: Fedora 34
 - Browser: N/A
 - Slidev version: 0.14.1

**Possible solution** 

This worked for me. The `line 2` in `@slidev/theme-default/setup/windicss.ts` should be 

```ts
import { defineWindiSetup } from '@slidev/types'
```
and not

```ts
import { defineWindiSetup } from 'packages/theme-seriph/node_modules/@slidev/types'
```

**Comment by antfu** [comment]
Em, nice catch up, thank you. Fixed in `@slidev/theme-default@v0.14.2`

**Comment by 15921483570** [comment]
now when i used a local theme `./theme`, it shows a similar error:

 `requireStack: ... windicss.ts`
